<launch>
	<node pkg="lab6" type="pursuit_avoid.py" name="pursuit_avoid">
		<param name="odom_topic" value="/pf/pose/odom"/>
	</node>
</launch>
